nucypher/policy/maps.py

```diff
@@ -324,21 +335,34 @@ def author(cls,
 
 class SignedTreasureMap(TreasureMap):
 
+    _BRAND = b'SM'
+
     def __init__(self, blockchain_signature=NOT_SIGNED, *args, **kwargs):
         self._blockchain_signature = blockchain_signature
         super().__init__(*args, **kwargs)
 
-    @classmethod
-    def splitter(cls):
-        return BytestringKwargifier(cls,
-                                    version=(bytes, len(cls._PREFIX)),
-                                    public_signature=Signature,
-                                    hrac=(bytes, HRAC_LENGTH),
-                                    message_kit=(UmbralMessageKit, VariableLengthBytestring),
-                                    blockchain_signature=65)
+    def __new__(cls, *args, **kwargs):
+        # TODO: This is a little too brittle-and-repeaty for my tastes; maybe we modify
+        # BSS to allow summing *before* the prefix(es)?
+
+        cls._splitters['unversioned'] = BytestringKwargifier(cls,
+                                                 blockchain_signature=65,
```

Comment:
`65` warrants a constant, e.g. in `nucypher.crypto.constants`. Maybe dependent on `Signature` size.